src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorTemplate.java

```diff
@@ -56,7 +54,8 @@ public record ErrorTemplate<T extends APIException>(
     String code,
     String title,
     String messageTemplate,
-    Optional<Integer> httpStatusOverride) {
+    Optional<Integer> httpStatusOverride,
+    EnumSet<ExceptionAction> exceptionActions) {
```

Comment:
rename exceptionFlags

and see below, how should these flags be used differenttly to the ones passed in toException ?